server/accounts.go

```diff
@@ -2970,6 +3024,17 @@ func (dr *DirAccResolver) Start(s *Server) error {
 			return fmt.Errorf("error setting up update handling: %v", err)
 		}
 	}
+	if _, err := s.sysSubscribe(accClaimsReqSubj, func(_ *subscription, _ *client, subj, resp string, msg []byte) {
+		if claim, err := jwt.DecodeAccountClaims(string(msg)); err != nil {
```

Comment:
I know we already discussed this many times, and I can't force you to change your style, but I find those `if/else` statements sometimes difficult to read. I personally find this easier:
```
claim, err := jwt.DecodeAccountClaims(string(msg))
if err != nil {
    respondToUpdate(s, resp, "n/a", "jwt update resulted in error", err)
    return
}
if err := dr.save(claim.Subject, string(msg)); err != nil {
    respondToUpdate(s, resp, claim.Subject, "jwt update resulted in error", err)
    return
}
respondToUpdate(s, resp, claim.Subject, "jwt updated", nil)
```